cmake/platform/linux/wayland.cmake

```diff
@@ -1,6 +1,21 @@
-set(PLATFORM_REQUIRED_DEPS OpenGl EGL Waylandpp LibDRM Xkbcommon)
```

Comment:
from the command line. if you build for wayland you have to decide to OpenGL or GLES. the result of a build should not be dependent on installed libs.